### PR TITLE
Add repology.org badges

### PR DESCRIPTION
**Pull request by Vifon**

### CODE REVIEW

**Comment by Vifon** [comment]
Thanks to @Zaeph for a few dozens of iterations that got squashed away.

File: README.md [hunk 1]
```diff
@@ -102,6 +122,7 @@ This also saves a list of all installed files to `install_log.txt`, which you ca
 use to uninstall ranger.
 
 
+
```
Comment (toonn) [inline README.md]: Uhm? Let's keep in mind to remove this again at some point? Triple blank line doesn't serve any purpose afaik?

Comment (Vifon) [reply README.md]: Rightio.

Comment (Zaeph) [reply README.md]: I'll fix that in a PR tomorrow.

**Pull request merged**